ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -71,7 +71,7 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
   implicit val system = getSystem
   implicit val mat = getMaterializer
   implicit val defaultPatience =
-    PatienceConfig(timeout = Span(10, Seconds), interval = Span(300, Millis))
+    PatienceConfig(timeout = Span(30, Seconds), interval = Span(600, Millis))
```

Comment:
@raboof I don't know why they're so slow on Travis. Of course, it has to do with the fact that these tests need to start and stop FTP servers but we should do fine tuning on these instances in order to configure the servers properly for working on Travis builds.